### PR TITLE
Support for downloading Analytics Adapters via http://prebid.org/download.html

### PR DESCRIPTION
**Pull request by anand-venkatraman**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [X] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change
In order to be able to build prebid.js from the Downloads page, this change would help specifying analytics adapters to include in the bundle. This change was requested as part of https://github.com/prebid/prebid.github.io/pull/132

### CODE REVIEW

**Comment by mkendall07** [comment]
can you modify this so that you can pass in a path to the file, the same way you can for standard adapters? Thanks!

File: analytics.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[
+  "appnexus",
+  "ga",
+  "pulsepoint",
+  "roxot",
+  "sharethrough_analytics"
```
Comment (mkendall07) [inline analytics.json]: let's remove these as this will change the default behavior of building with all the analytics adapters.

Comment (anand-venkatraman) [inline analytics.json]: Removed all the analytics adapter entries in the file. Though I think, it is probably a good idea to bundle all of them in the default build.

**Comment by mkendall07** [comment]
@snapwich @bretg 
Are you ok with this breaking change for including analytics adapters in the build - as a interim solution before the major refactor?

**Comment by snapwich** [comment]
yeah this is fine with me.

**Comment by mkendall07** [comment]
LGTM.

**Comment by ptomasroos** [comment]
Docs are outdated on building from local checkout repo.
Can't get any analytics into the build anymore. Pretty sure this PR killed it.

Plus there are no tests covering the usage of analytics.json in the build as far as I can tell

**Comment by ptomasroos** [comment]
New syntax is 
```
gulp build --adapters adapters.json --analyticsAdapters analytics.json
```

With an array defined in analytics.json

**Pull request merged**